grease/src/Grease/Macaw/Arch.hs

```diff
@@ -245,6 +246,10 @@ data ArchContext arch = ArchContext
   -- Currently, x86-64 is the only supported architecture that does something
   -- non-trivial here. All other architectures can simply leave the stack
   -- pointer by implementing @'_archOffsetStackPointerPostCall' = 'pure'@.
+  , _archABIParams :: [C.Some (MC.ArchReg arch)]
+  -- ^ The ordered integer and pointer ABI registers for the architecture. I
+  -- TODO: In the future this should handle things like stack params https://github.com/GaloisInc/grease/issues/260
+  -- TODO: In the future this should handle multiple calling conventions. https://github.com/GaloisInc/grease/issues/261
```

Comment:
https://galoisinc.github.io/grease/dev/style.html#numbered-todos
```suggestion
  -- TODO(#260): In the future this should handle things like stack params.
  -- TODO(#261): In the future this should handle multiple calling conventions.
```
We have a script that checks for stale TODOs in CI, but it only works for this format. Plus, this is more concise.